packages/cli/src/lib/redis/api.ts

```diff
@@ -71,11 +71,20 @@ export default (app: string, database: string | undefined, json: boolean, heroku
 
   return {
     request<T>(path: string, method: HttpVerb = 'GET', body = {}) {
-      const headers = {Accept: 'application/json'}
+      const headers = {Accept: 'application/vnd.heroku+json; version=3'}
       if (process.env.HEROKU_HEADERS) {
         Object.assign(headers, JSON.parse(process.env.HEROKU_HEADERS))
       }
 
+      // don't send an empty body, Cloudfront will reject the request
+      if (Object.keys(body).length === 0 && method === 'GET') {
```

Comment:
We want to always remove the body no matter what if method is 'GET', right? Not just when it's empty.
https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/RequestAndResponseBehaviorCustomOrigin.html#RequestCustom-get-body